packages/aragon-wrapper/src/rpc/handlers/external.js

```diff
@@ -27,11 +28,13 @@ export async function intent (request, proxy, wrapper) {
     ...params
   ] = request.params
 
-  const transactionPath = await wrapper.getExternalTransactionPath(
-    address,
-    methodJsonDescription,
-    params
-  )
+  const transactionPath = addressesEqual(address, wrapper.aclProxy.address)
+    ? await wrapper.getACLTransactionPath(methodJsonDescription.name, params)
```

Comment:
Actually, I'm wondering if this would make sense to move this logic to `getExternalTransactionPath()`, since the ACL is always an "external" destination to apps running against the RPC.

Reply:
Yes, it has total sense.